Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,132 @@ Meaning of columns:
 | Unit test | does OIP have unit tests for this entity; denoted by symbol, see [legend above](#legend) |
 | Issue | Link to corresponding open issue(s) |
 
-| Entity                | Covered              | Converter                  | Unit test             | Issue |
-|-----------------------|----------------------|----------------------------|-----------------------|-------|
-| `IfcCartesianPoint`   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
-| `IfcPointOnCurve`     | :x:                  | -                          | -                     | [#359](https://www.github.com/tumcms/Open-Infra-Platform/issues/359) |
+| Entity                                | Covered              | Converter                  | Unit test             | Issue |
+|---------------------------------------|----------------------|----------------------------|-----------------------|-------|
+| `IfcAdvancedBrep`                     | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedBrepWithVoids`            | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedFace`                     | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcAlignmentCurve`                   | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcAnnotationFillArea`               | :x:                  | -                          | -                     | -     |
+| `IfcAxis1Placement`                   | :x:                  | `PlacementConverter`       | :heavy_check_mark:    | -     |
```

Comment:
How can we have a unit test for an entity that we do not support? This doesn't make sense ...

Reply:
Resolved 3972ec7